crates/wasm-pkg-loader/src/source/mod.rs

```diff
@@ -6,7 +6,6 @@ use std::cmp::Ordering;
 
 use crate::{Error, PackageRef, Release};
 
-pub mod local;
```

Comment:
Why is this removed?

Reply:
I hadn't seen any examples or anything around that showed this and the loader code so I had assumed this was just an artifact for testing. Can add back

Reply:
Should the type name for this just be "local" or was there something else you had in mind?

Reply:
Yeah this was requested in a registry meeting to make sure we can handle different kinds of sources. I think "local" is fine in this context.